GEMINI.md

```
@@ -0,0 +1,98 @@
+*   Node.js 20+ & Yarn
+*   Go 1.21+
+*   Python 3.11+ (for AI services)
+
+### Quick Start (Full Platform)
+To start the entire platform locally using Docker Compose:
+
+```bash
+cd docker
+make up
+```
+*   **Frontend:** http://localhost:3001
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)